lib/improver/tests/blending/weights/test_ChooseWeightsLinear.py

```diff
@@ -571,6 +624,17 @@ def setUp(self):
                                           "weights": [0, 1, 1, 0],
```

Comment:
Please can you change "self.plugin_cube" to "self.plugin" throughout this class.